src/index.html

```diff
@@ -11,6 +11,28 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form data-qa="big" class="search-form">
+      <div class="search-form__input-wrapp">
+        <span class="search-form__icon search-form__icon--big"></span>
```

Comment:
Do you really need extra divs? Better use label instead. 
Also, you can set the background image directly to the input

Reply:
background-image was redefined when I changed only background on focus(